README.md

```diff
@@ -290,6 +290,11 @@ values =
 
 This configuration is in the section: `[bumpversion:file:…]`
 
+Note: The configuration file format requires each section header to be
+unique. If you want to process a certain file multiple times,
+you may append an underscore and a description to the `file` keyword:
+`[bumpversion:file_my_suffix:…]`.
```

Comment:
This isn't very clear to me. Wouldn't it be clearer to add another group to it? Like `[bumpversion:file:path/to/file:description]`?

Reply:
That would break existing file names which contain a colon.  Apparently a POSIX path is 'a collection of bytes with slashes having a special meaning' so in theory, anything goes.